lblod/app-digitaal-loket
# Feature/mandataris status code

Sets the status of all eredienstmandarissen in the database to status EFFECTIEF. The frontend has been updated so it sets the status by default when a new mandatee is created.

REF: DL-5090

To test:
- frontend branch: https://github.com/lblod/frontend-loket/pull/309
- setup stack and checkout this branch.
- wait for migration to finish
- run the following query
```
PREFIX ere: <http://data.lblod.info/vocabularies/erediensten/> 

SELECT (count(?mandataris) as ?count) WHERE {
  ?mandataris a ere:EredienstMandataris.
  FILTER NOT EXISTS {
    ?mandataris <http://data.vlaanderen.be/ns/mandaat#status> ?status
  }
}
```
Result should be 0 as all eredienstMandatarissen have to have a status

- run frontend on the `chore/worship-status` branch
- login as eredienst bestuur and create new mandataris
- capture the uuid of the newly created mandataris

run the following query

```
PREFIX mu: <http://mu.semte.ch/vocabularies/core/>

SELECT * WHERE {
  ?mandataris mu:uuid "insert uuid here";
  ?p ?o
}
```
Verify that the status triple is present

Note that the status is NOT shown in frontend-loket but it is shown in frontend OP
